src/js/view/controls/settings-menu.js

```diff
@@ -204,6 +203,10 @@ export function setupSubmenuListeners(settingsMenu, controlbar, viewModel, api)
         }
     }, settingsMenu);
 
+    model.change('playbackRateControls', () => {
```

Comment:
Does this needs to be invoked immediately? `model.change` is vs `model.on('change:...` which is only invoked after a change to the current value.

Reply:
Ooo nice, will update